orchagent/bulker.h

```diff
@@ -40,6 +32,13 @@ typedef sai_status_t (*sai_bulk_set_outbound_routing_entry_attribute_fn) (
         _In_ sai_bulk_op_error_mode_t mode,
         _Out_ sai_status_t *object_statuses);
 
+typedef sai_status_t (*sai_bulk_set_inbound_routing_entry_attribute_fn) (
```

Comment:
Curious about this change, does order in which these are defined matter?

Reply:
No, the order doesn't matter. I just thought of keeping pa_validation and outbound_ca_to_pa entries together as they correspond to APP_DB DASH_VNET_MAPPING_TABLE.